### ISSUE TITLE
37451059: Regression: compiling a macOS framework with the old build system produces Invalid bitcode signature

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#19336]
#### Description

Summary:
This is a duplicate of radar #37450480

This is a regression from Xcode 8.2.1, where it linked warning-free.
It does work with the new build system, however we’re blocked on rdar://35488529 and thus still use the old build system for all builds across platforms.

Steps to Reproduce:
Unsure what to deliver to reproduce. This cannot be reproduced in a sample app.

Expected Results:
Compile same as Xcode 9.2.1 or new build system.

Actual Results:
Invalid bitcode signature linker error

Version:
Xcode 9.3b2

Notes:
I also added -v as per Ankit Aggarwal’s comment on Twitter, but it did not yield any more useful data. (https://twitter.com/aciidb0mb3r/status/962743256571265025)

-
Product Version: Xcode 9.3b2
Created: 2018-02-12T07:30:49.555620
Originated: 2018-02-12T13:00:00
Open Radar Link: http://www.openradar.me/37451059
